msktutil.h

```diff
@@ -118,6 +118,18 @@
 #define MS_KERB_ENCTYPE_AES256_CTS_HMAC_SHA1_96 0
 #endif
 
+#define DEFAULT_MS_KERB_ENCTYPES \
+    MS_KERB_ENCTYPE_RC4_HMAC_MD5 | \
+    MS_KERB_ENCTYPE_AES128_CTC_HMAC_SHA1_96 | \
+    MS_KERB_ENCTYPE_AES256_CTS_HMAC_SHA1_96
+
+#define ALL_MS_KERB_ENCTYPES \
+    MS_KERB_ENCTYPE_DES_CBC_CRC | \
+    MS_KERB_ENCTYPE_DES_CBC_MD5 | \
+    MS_KERB_ENCTYPE_RC4_HMAC_MD5 | \
```

Comment:
Can we have it as MS_KERB_ENCTYPE_DES_CBC_CRC | \
     MS_KERB_ENCTYPE_DES_CBC_MD5 | \
    DEFAULT_MS_KERB_ENCTYPES ?

Reply:
Maybe even introducing DES_MS_KERB_ENCTYPES ? For reuse in msktldap.cpp ?

Reply:
This makes sense and reduces duplication. Will change `ALL_MS_KERB_ENCTYPES` and introduce `MS_KERB_DES_ENCTYPES`.

Reply:
Done.